src/Microsoft.Diagnostics.ExtensionCommands/EEHeap.cs

```diff
@@ -0,0 +1,506 @@
+﻿using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Runtime;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [Command(Name = "eeheap", Aliases = new string[] { "EEHeap" }, Help = "Displays information about native memory that CLR has allocated.")]
+    public class EEHeap : CommandBase
```

Comment:
The convention I've been using for command class and file names is "EEHeapCommand".